embassy-stm32/src/time_driver.rs

```diff
@@ -367,6 +362,11 @@ impl RtcDriver {
                 self.next_period();
             }
 
+            // Half overflow
+            if sr.ccif(0) {
+                self.next_period();
+            }
+
```

Comment:
This will cause the tick rate to be double-time.

Reply:
You are correct.

I saw you removed it and noted that change when it conflicted, but it looks like I accepted the wrong side.

Pushed a commit to fix this.